test/stripe/balance_transaction_test.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require ::File.expand_path("../test_helper", __dir__)
+
+module Stripe
+  class BalanceTransactionTest < Test::Unit::TestCase
+    should "be listable" do
+      BalanceTransactions = Stripe::BalanceTransaction.list
```

Comment:
Is this naming on purpose? Usually camel-case is constant naming in Ruby, so this should probably be `balance_transactions` and the instance below singular `balance_transaction`.

Reply:
Ah no, just a bad string replace, nice catch, fixed!